readme.md

```diff
@@ -206,6 +215,7 @@ into markdown and saved to readme.md.
 * Ensure compliance with Micropub spec
 * Update composer dependencies and include PHPUnit as a development dependency
 * Add nag notice for http domains and the option to diable with a setting
+* `MICROPUB_LOCAL_AUTH` is now deprecated in favor of a filter
```

Comment:
this PR does more than deprecate it; it removes it entirely. that's fine, it's not a compatibility concern, just checking on language.